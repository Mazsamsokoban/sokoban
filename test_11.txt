loadtest 11
listworkers
showstate box
move w2 right
showmaze
listworkers
showstate box