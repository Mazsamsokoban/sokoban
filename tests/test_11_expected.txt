f0:w1 f1:w2 f2:box bf:null 

name:w1
field:f0
direction:Neutral
points:1
pushforce:10.0

name:w2
field:f1
direction:Neutral
points:0
pushforce:10.0

name:box
field:f2
direction:Neutral
owner:w1
pusher:null

f0:w1 f1:null f2:w2 bf:box 

name:w1
field:f0
direction:Neutral
points:1
pushforce:10.0

name:w2
field:f2
direction:Right
points:0
pushforce:10.0

name:box
field:bf
direction:Right
owner:w1
pusher:w2

